tests/BenchmarkDotNet.Tests/Validators/FakeSdkProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿using BenchmarkDotNet.Validators;
+using System.Collections.Generic;
+
+namespace BenchmarkDotNet.Tests.Validators
+{
+    public class FakeSdkProvider : ISdkProvider
+    {
+        private readonly string[] installedSdks;
+        public string CustomDotNetCliPath { get; set; }
+
+        public FakeSdkProvider(string[] installedSdks, string customDotNetCliPath = "")
+        {
+            this.installedSdks = installedSdks;
+            this.CustomDotNetCliPath = customDotNetCliPath;
+        }
+
+        public IEnumerable<string> GetInstalledSdks()
+        {
+            if (!string.IsNullOrEmpty(CustomDotNetCliPath))
+            {
+                return installedSdks;
+            }
```

Comment:
I don't understand this check. Both paths return the same thing.

Reply:
Oops. Me neither. 😄